bin/tx-prover/src/proxy/metrics.rs

```diff
@@ -0,0 +1,49 @@
+use lazy_static::lazy_static;
+use prometheus::{
+    register_histogram, register_int_counter, register_int_counter_vec, register_int_gauge,
+    Histogram, IntCounter, IntCounterVec, IntGauge,
+};
+
+lazy_static! {
```

Comment:
Is this usage not covered under the `std` replacement yet? (to be clear - I'm unsure what `std` covers at the moment).

Reply:
I did the test and works perfectly fine with `LazyLock`. I'm fully changing towards it.